### DISCUSSION TITLE
Will VVenC further optimize the SCC mode?

### DISCUSSION THREAD

**Discussion by 9reyson** [fraunhoferhhi/vvenc#56] [Q&A]
Hi!

The VVenC encoder has implemented TS and BDPCM. I want to know whether VVenC will further optimize the performance of the screen sequences. For example, supporting IBC, Palette, and HashME tools.

Best regards,
Greyson

**Comment by adamjw24** [comment]
Dear Greyson,

We are looking into supporting IBC in the next version. HashME is just an extension of the IBC search algorithm. Palette is a coding mode for the 444 profile, which we do not support and do not plan on supporting soon.

Best regards,
Adam

**Reply by 9reyson** [reply]
Dear Adam,

Thanks a lot for your answer! It's very useful for me!

Best regards,
Greyson
